admin/authentication/dfe-signon-strategy.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+const config = require('../config')
+const { Strategy, Issuer } = require('openid-client')
+const logger = require('../services/log.service').getLogger()
+const asyncRetry = require('login.dfe.async-retry')
+
+const initSignOn = async () => {
+  Issuer.defaultHttpOptions = { timeout: 10000 }
```

Comment:
Are the units milliseconds?

Reply:
taken verbatim from provided example, will formalise this soon.

Reply:
now a formal setting with unit postfix